packages/taquito-michel-codec/.eslintrc.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
Just curious, why this is removed?

Reply:
Because ESLint can (and should in case of a monolithic project) pick its config from the root

Reply:
thanks, will make the change in the other packages